Assets/MRTK/Examples/Demos/EyeTracking/General/Scripts/TargetBehaviors/OnLoadStartScene.cs

```diff
@@ -16,9 +22,27 @@ public class OnLoadStartScene : MonoBehaviour
         [Tooltip("Name of the scene to be loaded when the button is selected.")]
         private string SceneToBeLoaded = "";
 
+        [SerializeField]
+        [Tooltip("Option to only load the scene if running on the HoloLens device.")]
+        private LoadOptions LoadOption = LoadOptions.LoadOnlyOnDevice;
+
         public void Start()
         {
+            if (LoadOption == LoadOptions.LoadOnlyOnDevice)
+            {
+                LoadOnDevice();
+            }
+            else
+            {
+                LoadNewScene();
+            }
+        }
+
+        private void LoadOnDevice()
+        {
+#if WINDOWS_UWP
```

Comment:
Should we use something like [`!Application.isEditor`](https://docs.unity3d.com/ScriptReference/Application-isEditor.html) to detect when we're on-device?

Reply:
Good idea! Thanks for that reminder.